### DISCUSSION TITLE
Metadata for entity User not found with HMR

### DISCUSSION THREAD

**Discussion by chiqui3d** [mikro-orm/mikro-orm#6264] [Q&A]
Every time I make a change and save, I get this error. If I edit something else, the error disappears, and then it happens again when I make another change. It seems to be something related to Nest.js HMR because I didn’t experience this issue before, but it was really slow back then. Now it’s faster, but I have to keep editing something to make it work each time.

Can someone help me figure this out?

**Comment by B4nan** [comment]
Can you share a reproduction?
